examples/applications/svm_gui.py

```diff
@@ -24,7 +24,7 @@
 matplotlib.use('TkAgg')
 
 from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg
-from matplotlib.backends.backend_tkagg import NavigationToolbar2TkAgg
+from matplotlib.backends.backend_tkagg import NavigationToolbar2Tk
```

Comment:
We still want this to work with matplotlib 2, so it should be,
```py
try:
    from matplotlib.backends.backend_tkagg import NavigationToolbar2Tk
except ImportError:
    from matplotlib.backends.backend_tkagg import (
          NavigationToolbar2TkAgg as NavigationToolbar2Tk
    )
```

Reply:
@rth Just did it the way you suggested with few other changes. Hope it suits the requirements !